### ISSUE TITLE
Accessibility violations in disabled table controls

### ISSUE THREAD

**Issue by caritaou** [elyra-ai/canvas#2555]
ref 4039

Disabled tables were fixed as part of https://github.com/elyra-ai/canvas/issues/2508 but controls using tables still have their own opacity overrides that needs to be removed, so that it will default to what was done in flexibleTable

- expression
- list
- readonlyTable
- selectColumns
- structurelisteditor
- textPanels
